www/javascript/site-content-slider.js

```diff
@@ -66,6 +68,8 @@ YAHOO.util.Event.onDOMReady(function ()
 		this.touch_start_y = null;
 		this.touch_end_x = null;
 
+		this.pageChangeEvent = new YAHOO.util.CustomEvent('pageChange', this);
```

Comment:
I think you might just be confused by the name of the event. This just fires when the "pager" changes to a new "page" - and has nothing to do with the actual page.